Dynamically test javascript URL string return values in frames

rwlbuis opened the pull request:
Test javascript URL string return values in direct and indirect (target) frame contexts.

annevk commented on html/browsers/browsing-the-web/navigating-across-documents/javascript-url-return-value-handling-dynamic.html:
```diff
@@ -0,0 +1,48 @@
+    }, 200)
+    document.body.appendChild(frame);
+    document.body.appendChild(href);
+    href.click();
```
Wouldn't this ruin the entire harness in user agents that don't use the `target` attribute?

rwlbuis replied:
I was not ware Firefox did not support it. Anyway latest update should be better as it splits up the frame scenario and the link target scenario.

annevk replied:
Firefox does, but Chrome/Safari don't, right?

rwlbuis replied:
Yes, that is correct, and I have a fix ready for WebKit/Safari.
I noticed an error in my change, should be fixed now.

rwlbuis committed:
Dynamically test javascript URL string return values in frames

Test javascript URL string return values in direct and indirect (target) frame contexts.
